x/ccv/provider/keeper/keeper.go

```diff
@@ -178,6 +178,50 @@ func (k Keeper) DeleteChainToChannel(ctx sdk.Context, chainID string) {
 	store.Delete(types.ChainToChannelKey(chainID))
 }
 
+// SetProposedConsumerChain sets consumer chainId in gov consumerAddition proposal in store
+// the consumer chainId is only added when the voting period for consumerAddition proposal
+// does not end.
+func (k Keeper) SetProposedConsumerChain(ctx sdk.Context, chainID string, proposalID uint64) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.ProposedConsumerChainKey(proposalID), []byte(chainID))
+}
+
+// GetProposedConsumerChain get the proposed chainID  in consumerAddition proposal.
+func (k Keeper) GetProposedConsumerChain(ctx sdk.Context, proposalID uint64) string {
```

Comment:
This method is unused

Reply:
this is only used in test to test `SetProposedConsumerChain`